files/zh-cn/web/css/adjacent_sibling_combinator/index.md

```diff
@@ -42,9 +42,7 @@ li:first-of-type + li {
 
 ### 结果
 
-- One
-- Two!
-- Three
+{{EmbedLiveSample("Examples", "100%", 100)}}
```

Comment:
```suggestion
{{EmbedLiveSample("示例", "100%", 100)}}
```

Reply:
🙏 I confirm that it works ~

![CleanShot 2022-10-02 at 17 11 41@2x](https://user-images.githubusercontent.com/11187239/193446959-cd9c5c25-08a9-467f-8d83-a8cc306663a4.png)